pyproject.toml

```diff
@@ -34,21 +34,46 @@ classifiers = [
 ]
 dependencies = [
     "orjson>=3.11.3",
-    "urllib3>=2.5.0",
+    "urllib3",
     "verlib2>=0.3.1",
 ]
 
 [dependency-groups]
 dev = [
     "certifi>=2025.10.5",
+    "coverage<8",
     "mypy<1.19",
     "poethepoet<1",
     "pytest<9",
-    "pytest-cov<8",
     "pytz>=2025.2",
     "ruff<0.15",
     "setuptools>=80.9.0",
     "stopit<1.2",
+    "urllib3<2.4",
```

Comment:
FYI: This can be removed after tackling https://github.com/crate/crate-python/issues/708.
```suggestion
```